packages/playwright-test/src/test.ts

```diff
@@ -23,6 +23,7 @@ class Base {
   title: string;
   _only = false;
   _requireFile: string = '';
+  _projectSetup: boolean = false;
```

Comment:
Since we are doing this, let's switch all `setupFiles.has(_requireFile)` checks to `_isProjectSetup`? Could be a follow up.

Reply:
will do in a follow-up